src/components/header_menu.jsx

```diff
@@ -35,13 +35,17 @@ const HeaderMenu = ({
   return (
     <header className={styles['header-menu']}>
       <WithBreakpoints hideOn={Breakpoints.Mobile}>
-        <AppLinker slug={Slugs.Home} href={homeHref}>
-          {({ href }) => (
-            <Link {...(!isPublic && { href })} className={styles['home-link']}>
-              <AppIcon app={Slugs.Home} alt={Slugs.Home} />
-            </Link>
-          )}
-        </AppLinker>
+        {homeHref ? (
```

Comment:
homeHref is always defined even with a 0 cookie user.  
`collabProvider.serviceClient.cozyClient.getStackClient().uri`

Is there an easy way to differentiate a cozy user previewing the page from a non cozy user? The token in the url makes real cozy users and non cozy users looking the same in the client

Reply:
Make a search on `isPublic` into the codebase, you'll find what you want, I think. You removed it in this component...

Reply:
I don't understand the meaning of `homeHref`. We should remove this concept and restore `isPublic` stuff.

Also, we should only use the SVG from the public folder for the both case. The only stuff that is changing is that we don't put the href to the link if we're in public. Like we did before.